durabletask/client.py

```diff
@@ -92,38 +92,46 @@ def new_orchestration_state(instance_id: str, res: pb.GetInstanceResponse) -> Or
 class TaskHubGrpcClient:
 
     def __init__(self, *,
-                 host_address: str | None = None,
+                 host_address: Union[str, None] = None,
                  log_handler=None,
-                 log_formatter: logging.Formatter | None = None):
+                 log_formatter: Union[logging.Formatter, None] = None):
         channel = shared.get_grpc_channel(host_address)
         self._stub = stubs.TaskHubSidecarServiceStub(channel)
         self._logger = shared.get_logger(log_handler, log_formatter)
 
-    def schedule_new_orchestration(self, orchestrator: task.Orchestrator[TInput, TOutput], *,
-                                   input: TInput | None = None,
-                                   instance_id: str | None = None,
-                                   start_at: datetime | None = None) -> str:
+    def schedule_new_orchestration(self, orchestrator: Union[task.Orchestrator[TInput, TOutput], str], *,
+                                   input: Union[TInput, None] = None,
+                                   version: Union[str, None] = None,
```

Comment:
Let's not include version in this API yet. We shouldn't need it.

Reply:
It fails here in this case: https://github.com/microsoft/durabletask-go/blob/91f53a8f8b7fa24a3cb348fb8290594f6ed1c641/backend/executor.go#L286

When testing with dapr, which is currently using v0.1.4 (https://github.com/dapr/dapr/blob/dbeea96710d38d08dcc5aaf6807914a7d697bed4/go.mod#L35), I have following code piece:
```
ctx, span := helpers.StartNewCreateOrchestrationSpan(ctx, req.Name, req.Version.Value, instanceID)
```

Reply:
I see. In that case, let’s fix it by sending an empty string as the version, but let’s not expose the version to our Python APIs.

Reply:
Done